README.md

```diff
@@ -1,10 +1,10 @@
-# jQuery Fungus Fighter
+# JavaScript Fungus Fighter
 
-Fight mutant fungus villains, with jQuery!
+Fight mutant fungus villains, with JavaScript!
 
 ![demo](demo.gif)
 
-It is you versus the Freaky Fungus, an evil mycological menace from mars. Use your jQuery skills to dispatch attacks against your enemy. To win, the Freaky Fungus must have 0 hit points (HP) before you are out of attack points (AP).
+It is you versus the Freaky Fungus, an evil mycological menace from ***Wisconsin***. Use your JavaScript skills to dispatch attacks against your enemy. To win, the Freaky Fungus must have 0 hit points (HP) before you are out of attack points (AP).
```

Comment:
As a native of Wisconsin, I can not endorse this.

More importantly, and perhaps too nitpicky, but I do think it's legit, I think some students may be confused by this.

Specifically because we refer to "state variables" a few lines later.